documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -6,48 +6,96 @@
 
 = Optimization proposals overview
 
-An _optimization proposal_ is a summary of proposed changes that would produce a more balanced Kafka cluster, with partition workloads distributed more evenly among the brokers. 
+An _optimization proposal_ is a summary of proposed changes that would produce a more balanced Kafka cluster, with partition workloads distributed more evenly among the brokers.
 Each optimization proposal is based on the set of xref:con-optimization-goals-{context}[optimization goals] that was used to generate it, subject to any configured xref:capacity-configuration[capacity limits on broker resources].
 
-An optimization proposal is contained in the `Status.Optimization Result` property of a `KafkaRebalance` custom resource. 
-The information provided is a summary of the full optimization proposal. 
-Use the summary to decide whether to:
+All optimization proposals are _dry runs_: you cannot approve a cluster rebalance without first generating an optimization proposal.
```

Comment:
I think this might be mis-leading. Better wording might be: 
```
All optimization proposals are _estimates_ of the impact of a proposed rebalance. You cannot approve a cluster rebalance without first generating an optimization proposal.
```

Reply:
Maybe we can mention you can approve or reject here. 

> All optimization proposals are _estimates_ of the impact of a proposed rebalance. You can approve or reject a proposal. You cannot approve a cluster rebalance without first generating the optimization proposal. There is no limit to the number of optimization proposals that can be generated.